packages/nextjs/app/builders/0x167142915AD0fAADD84d9741eC253B82aB8625cd/page.tsx

```diff
@@ -0,0 +1,126 @@
+"use client";
+
+import Image from "next/image";
+import { Address } from "~~/components/scaffold-eth";
+
+const socialLinks = {
+  github: "nnennaokoye",
+  twitter: "_0xNina",
+  linkedin: "nnennaokoye",
+  instagram: "_ninacodes",
+};
+
+export default function BuilderPage() {
```

Comment:
Please name your page with something more unique, like your name or nickname.
And you might want to consider typing your page as NextPage since we are using TypeScript, as this ensures better integration and adds clarity for future maintainers by indicating that the component is a Next.js page.